### ISSUE TITLE
Problem controlling Zwave COLOR_SWITCH, with fix

### ISSUE THREAD

**Issue by NickWaterton** [shbatm/PyISY#17]
I just added a zwave device that has an RGB light (HSM200) I figured out the REST command to control the RGB is `/rest/nodes/ZW005_186/cmd/DON?GV2=0&GV3=0&GV4=255` where GV2,3,4 is r,g,b respectively.

Sending this using `node.send_cmd('DON?GV2=0&GV3=0&GV4=255')` does not work, as the string is then url encoded by `req_url = self.isy.conn.compile_url(req)`, and this fails.

My solution is to modify `send_cmd` to allow the addition of the query parameter like this:
```
    def send_cmd(self, cmd, val=None, uom=None, query=None):    #NW added query
        """Send a command to the device."""
        value = str(val) if val is not None else None
        _uom = str(uom) if uom is not None else None
        req = ["nodes", str(self._id), "cmd", cmd]
        if value:
            req.append(value)
        if _uom:
            req.append(_uom)
        req_url = self.isy.conn.compile_url(req, query)    #added query here
        if not self.isy.conn.request(req_url):
            self.isy.log.warning(
                "ISY could not send %s command to %s.",
                COMMAND_FRIENDLY_NAME.get(cmd),
                self._id,
            )
            return False
        self.isy.log.info(
            "ISY command %s sent to %s.", COMMAND_FRIENDLY_NAME.get(cmd), self._id
        )

        # Calculate hint to use if status is updated
        hint = self.status._val
        if cmd in ["DON", "DFON"]:
            if val is not None:
                hint = val
            elif "OL" in self._aux_properties:
                hint = self._aux_properties["OL"].get("value")
            else:
                hint = 255
        elif cmd in ["DOF", "DFOF"]:
            hint = 0
        self.update(UPDATE_INTERVAL, hint=hint)
        return True
```

This means that I can now send the required command like this:
```
val = value.split('?')
target_node.send_cmd(val[0], query = json.loads(val[1]))
```
where value is a string `'DON? {"GV2":0, "GV3":0, "GV4":255}` so I can now set my RGB light correctly.

This is probably useful if there are other query strings that need to be sent to nodes, especially zwave nodes, and doesn't interfere with any other commands.

**Comment by shbatm** [comment]
Available now in `PyISY_beta-2.0.0.dev86`

**Comment by shbatm** [comment]
@NickWaterton  I'm working on adding this to Home Assistant, would you mind sending me the XML dump for this node? (`/rest/nodes/ZW005_186`)?
